application.py

```diff
@@ -18,33 +18,75 @@
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 db.init_app(app)
 
-
 with app.app_context():
     db.create_all()
```

Comment:
create_all is a function to create tables, there is no need to run the method each time the flask app is run